.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for the Pull Request!
+
+Please review [Contribution Guide](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md) before opening your first Pull Request.
+
+To ensure your Pull Request is reviewed and accepted quickly, please refer to our [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines).
+
+--->
+
+# Description
+
+<!--- Describe your changes in detail -->
+
+<!--- Consider: Why is this change required? What problem does it solve? -->
+
+<!--- Include screenshots if appropriate --->
+
+## Issue number and link
+
+<!--- If it fixes an open issue, link to the issue here -->
+
+<!--- Consider: If suggesting a new feature or change, discuss it in an issue first. -->
+
+## Testing plan
+
+<!--- Describe in detail how you tested your changes -->
+
+# Author checklist
+
```

Comment:
Maybe an extra comment should be included here for how to handle items that don't apply to a given PR? Should they be crossed out? deleted? checked but include `N/A`?
Just a one line suggestion to avoid confusion

Reply:
I'd rather have people answer in different ways than not answer at all– And I think checkboxes are about as low effort as they can get.

If that's too confusing though, I think we'll be able to tell from the submitted PR descriptions and we can alter it later. What do you think?

Reply:
We can leave it out for now if you want. I was just trying to get ahead of someone asking "What do I put if one doesn't apply? do I check it or not?"